src/main/java/org/spongepowered/api/command/conversation/ConversationArchetype.java

```diff
@@ -0,0 +1,351 @@
+     */
+    boolean allowsCommands();
+
+    /**
+     * Gets the default external chat handler type for all {@link Conversant}s
+     * within the conversation.
+     *
+     * @return The default external chat handler type for all {@link Conversant}s
+     *     within the conversation
+     */
+    ExternalChatHandlerType getDefaultChatHandlerType();
+
+    /**
+     * Gets the handlers that handles the end of the {@link Conversation}.
+     *
+     * <p>The collection is implementation dependent, but is
+     * generally immutable or a copy.</p>
+     *
+     * @return All ending handlers
+     */
+    Collection<EndingHandler> getEndingHandlers();
+
+    /**
+     * Gets the banner that will be sent before each question prompt
+     * if title is set.
+     *
+     * <p>This is a combination of the padding and title set.</p>
+     *
+     * @return The question banner, if available
+     */
+    Optional<Text> getBanner();
+
+    /**
+     * Gets the header that occurs before each question, or after
+     * the banner if the the banner is set.
+     *
+     * <p>This is completely optional and is often not set.</p>
+     *
+     * @return The question header, if set
+     */
+    Optional<Text> getHeader();
+
+    /**
+     * Gets the title used to create the header, if set.
+     *
+     * @return The title of the conversation, if available
+     */
+    Optional<Text> getTitle();
+
+    /**
+     * Gets the padding used to create the header, if set.
+     *
+     * @return The padding of the conversation, if available
+     */
+    Optional<Text> getPadding();
+
+    /**
+     * Gets the message sent to {@link Conversant}s when they
+     * try to use a command in a conversation and that is disabled.
+     *
+     * @return The no command usage message
+     */
+    Text getNoCommandUsageMessage();
+
+    /**
+     * Gets the handler which handles input to see if it should end
+     * the {@link Conversation}.
+     *
+     * @return The cancelling handler
+     */
+    CancellingHandler getCancellingHandler();
+
+    /**
+     * Starts the conversation for the specified conversants.
+     *
+     * <p>Generally only one conversant is set, as each question is completed
+     * after just one responds, but multiple are allowed to support the use case
+     * of multiple participants. If you want them to be in their own
+     * conversation, use the archetype to simply start another one.</p>
+     *
+     * @param plugin The plugin starting the conversation
+     * @param conversants The conversants to start the conversation for
+     * @return The conversation that was started, if created successfully
+     * @throws IllegalArgumentException If the object passed in is not
+     *     a plugin instance
+     */
+    Optional<Conversation> start(Object plugin, Conversant... conversants);
+
+    /**
+     * Gets a builder based off of this archetype's values.
+     *
+     * @return The builder
+     */
+    Builder toBuilder();
```

Comment:
No... Add a method `from(ConversationArchetype archetype)` in the builder instead.